cms/djangoapps/contentstore/views/course.py

```diff
@@ -984,6 +947,7 @@ def settings_handler(request, course_key_string):
                 'is_entrance_exams_enabled': is_entrance_exams_enabled(),
                 'self_paced_enabled': self_paced_enabled,
             }
+            settings_context.update(get_context_for_settings_handler(course_key))
```

Comment:
This is the cleanest way for us to inject our values.
Note: we're adding ours after the dict has been created,
as to give ourselves higher priority.